bin/CodeChecker.py

```diff
@@ -78,37 +74,33 @@ def signal_handler(sig, frame):
             if len(sys.argv) > 1:
                 first_command = sys.argv[1]
                 if first_command in subcommands:
-                    LOG.debug("Supplied an existing, valid subcommand: " +
-                              first_command)
 
                     # Consider only the given command as an available one.
                     subcommands = [first_command]
 
             for subcommand in subcommands:
-                LOG.debug("Creating arg parser for subcommand " + subcommand)
 
                 try:
                     libhandlers.add_subcommand(subparsers, str(subcommand))
                 except (IOError, ImportError):
-                    LOG.warning("Couldn't import module for subcommand '" +
-                                subcommand + "'... ignoring.")
+                    print("Couldn't import module for subcommand '" +
```

Comment:
Why are we replacing logs with prints in this file?

Reply:
The log configurations and log levels can be set only later where the arguments are parsed to keep the current behavior. Until the loggers are not set they can not be used.